pkg/resources/secret_common.go

```diff
@@ -0,0 +1,112 @@
+package resources
+
+import (
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/schemas"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+var secretCommonSchema = map[string]*schema.Schema{
+	"name": {
+		Type:             schema.TypeString,
+		Required:         true,
```

Comment:
Snowflake doesn't support RENAME TO, so this should be ForceNew: true.

Reply:
added ForceNew